package.json

```diff
@@ -78,8 +79,7 @@
 		"mocha": "^9.1.1",
 		"nyc": "^15.1.0",
 		"prettier": "^2.4.1",
-		"ts-expose-internals": "=4.5.5",
```

Comment:
Moving ts-expose-internals to its new recommended install procedure. Doing this means that the line removed from `tsconfig-base.json` is redundant.